sqlx-runner/insert_exec_test.go

```diff
@@ -106,35 +108,78 @@ func TestInsertReal(t *testing.T) {
 		InsertInto("people").
 		Columns("name", "email").
 		Record(&person).
-		Returning("id", "created_at").
+		Returning("id", "email", "created_at").
 		QueryStruct(&person)
-	assert.NoError(t, err)
+	require.NoError(t, err)
```

Comment:
should these be asserts instead of requires? It seems like that was the pattern.

Reply:
i think require since the tests are irrelevant if there is an error there -- so basically we want to stop at that point.